src/gui/sdlmain.cpp

```diff
@@ -1829,12 +1890,55 @@ static std::string NormalizeConfValue(const char *val)
 	return pref;
 }
 
+static bool DetectResizableWindow()
+{
+#if C_OPENGL
+	if (sdl.desktop.want_type != SCREEN_OPENGL) {
+		LOG_MSG("MAIN: Disabling resizable window, because it's not "
+		        "compatible with selected sdl.output");
+		return false;
+	}
+
+	assert(control);
+	const Section *rs = control->GetSection("render");
+	assert(rs);
+	const std::string sname = rs->GetPropValue("glshader");
+
+	if (sname != "sharp" && sname != "none") {
+		LOG_MSG("MAIN: Disabling resizable window, because it's not "
+		        "compatible with selected render.glshader");
```

Comment:
Suggest adding to the message the user's incompatible glshader along with the two glshaders that are compatible.